src/locales/ko/mystery-encounters/dancing-lessons-dialogue.json

```
@@ -1,7 +1,7 @@
 {
   "intro": "{{oricorioName}}[[가]] 외로이 혼자 춤을 추고 있다.\n주변에 파트너도 없는 것 같다!",
-  "title": "댄스 댄스 레볼루션",
-  "description": "{{oricorioName}}[[는]] 공격적이진 않아 보입니다.\n오히려 슬퍼 보이기까지 하네요.\n\n그저 함께 춤출 누군가를\n원하고 있는지도 모릅니다….",
+  "title": "댄스 레슨",
+  "description": "{{oricorioName}}[[는]] 공격적이진 않아 보입니다.\n오히려 풀이 죽은 것 같네요.\n\n아마도 함께 춤출 누군가를\n원하고 있는지도 모릅니다….",
```

```suggestion
  "description": "{{oricorioName}}[[는]] 공격적이진 않아 보입니다.\n오히려 의기소침해 보이네요.\n\n아마도 함께 춤출 누군가를\n원하고 있는지도 모릅니다….",
```
despondent의 뜻을 생각해 본다면 풀이 죽었다고 하는 것도 옳지만, 풀죽게 만든 사람이나 포켓몬 없이 자기 혼자 꿍해 있는 느낌이라 의기소침하다고 하는 게 어떨까 제안합니다...